### DISCUSSION TITLE
changelog theme (purchased) - background color for dark theme not working?

### DISCUSSION THREAD

**Discussion by Caluix** [GetPublii/Publii#1846] [Help]
Background color for dark theme isn't working.

I set this

![image](https://github.com/user-attachments/assets/fc7ff2ab-31cf-4eab-90d1-b8bfa6fd3779)

But the background color of the body element is:

![image](https://github.com/user-attachments/assets/5a90023e-ca4d-44e1-9958-1d3c6862ffd4)

Maybe it's my fault but I don't understand how to set black background. Thanks!

**Comment by bobmitro** [comment]
Hi, this is a Premium theme. Please use the dedicated Premium support. Log in to [https://marketplace.getpublii.com/](https://marketplace.getpublii.com/), go to the dashboard, and create a ticket.

**Comment by Caluix** [comment]
Hi! I tried but my support access is expired and get me here.

![image](https://github.com/user-attachments/assets/deef2df0-3775-4a97-a2fc-0c9b7ae74387)

Don't know why as I bought the theme few days ago. 🤔

**Comment by Caluix** [comment]
At the moment I made a workaround with the custom-css option. :)
